client/src/rpb/rpb.scss

```diff
@@ -93,35 +97,32 @@ li.select-list-item {
   }
 }
 
-
-.pagination-container {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
-  & > .pagination > li > span {
-    padding: 10px 16px;
-  } 
-}
-
-.row-opition-content-search > div > div > .input-lg {
-  height: 38px;
-  line-height: 0.9;
+.full-width {
+  width: 100%;
 }
 
 .rpb-separator {
   border-bottom: 1px solid $separatorColor;
 }
-
-.rpb-share-options {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-evenly;
-
-  & > button {
-    background-color: Transparent;
-    border: none;  
-    &:active {
-      transform: scale(1.15);
-    }
+.rpb-heading-utils {
+  width: 20px;
+  height: 20px;
+}
+button.rpb-heading-utils {
+  background-color: Transparent;
+  border: none;
+  margin-right: 5px;
+  padding: 0;
+
+  &:active {
+    transform: scale(1.15);
   }
-}
\ No newline at end of file
+}
+.group_filter_dropdown {
+  display: flex;
+  flex-direction: column;
+}
+.normal-rd-label {
```

Comment:
What's "rd"? I could go look where it's used to figure out, but it's nicer to spell things out in the class name itself. If the place it's used was deleted, this would just be a mystery class.